src/app/file-upload/file-upload.component.ts

```diff
@@ -13,7 +13,8 @@ export class FileUploadComponent {
   public dragDropClass = 'dragarea';
   @Input() fileExt = 'jpg, jpeg, gif, png, bmp, doc, docx, xls, xlsx, ppt, pptx, pdf, txt';
   @Input() maxFiles = 50;
-  @Input() maxSize = 300; // in MB
+  @Input() maxSize = 3000; // in MB
```

Comment:
While this changes the default input value, it's not the only area that needs updating.

Reply:
Found and updated maxSize in 3 separate upload forms.